models/scheduler.go

```diff
@@ -371,6 +374,39 @@ func (c *Scheduler) GetResourcesRequests() map[AutoScalingPolicyType]int64 {
 	return res
 }
 
+// SavePodsMetricsUtilizationPipeAndExec set a sorted set on redis with the usages of all pods from a scheduler
+func (c *Scheduler) SavePodsMetricsUtilizationPipeAndExec(
+	redisClient redisinterfaces.RedisClient,
+	metricsClientset metricsClient.Interface,
+	mr *MixedMetricsReporter,
+	metric AutoScalingPolicyType,
+	rooms []*Room,
+	usages []float64,
+) error {
+
+	if len(rooms) == 0 || len(usages) == 0 {
+		return nil
+	}
+
+	pipe := redisClient.TxPipeline()
+	for i, room := range rooms {
+		pipe.ZAdd(
+			GetRoomMetricsRedisKey(c.Name, string(metric)),
+			redis.Z{Member: room.ID, Score: float64(usages[i])},
```

Comment:
are we checking somewhere that `len(rooms) == len(usages)`?

Reply:
No. It can't be different but I'm gonna make it send only one struct with usage and room in it

Reply:
Yep, I saw that this shouldn't happen but I'd rather have an array of structs instead of two arrays.